shared/hedvig-ui/animations/ease-in.tsx

```diff
@@ -0,0 +1,15 @@
+import styled, { keyframes } from 'react-emotion'
+
+const fadeIn = (max) =>
+  keyframes({
+    from: { opacity: 0, transform: 'translateY(2%)' },
+    to: { opacity: max, transform: 'translateY(0)' },
+  })
+
+export const EaseIn = styled('div')<{ delay?: string }>(
```

Comment:
I think 🤔 we should use the `` syntax for styled components. @palmenhq would have to correct me on that though.
```suggestion
export const EaseIn = styled('div')<{ delay?: string }>`
```